_harp/apply/london.ejs

```diff
@@ -0,0 +1,17 @@
+<div class="section-copy">
+	<div class="container apply-page">
+		<h1 align='center'>LONDON</h1>
+
+		<p align='center'>Apply today for our next course: 23 October, 2017 - February 9, 2018</p>
+
+		<div align='center'>
+		  <a href="https://goo.gl/forms/CERu8qvH6kjL7pZG3" target="_blank"><button class="button-three">APPLY</button></a>
+		</div>
+
+		<%- partial("_london.md") %>
+
+		<div align='center'>
+		  <a href="https://goo.gl/forms/CERu8qvH6kjL7pZG3" target="_blank"><button class="button-three">APPLY</button></a>
```

Comment:
My thinking was that you don't want to have to scroll all the way to the bottom for the Apply button. Especially since it's been a couple of clicks already since they first clicked on the "Apply" tab.

But you also don't want them to refrain from clicking the button, choose to read through all the text, and then have to scroll all the way back up again.

But I agree it's repetitive :thinking: 
Get rid of the 2nd one?